exporter.go

```diff
@@ -179,8 +180,10 @@ func (e *Exporter) ScrapeHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	// get rid of username/password info in "target" so users don't send them via http
-	u.User = nil
+	if u.Scheme != "rediss" {
+		// get rid of username/password info in "target" so users don't send them via http
```

Comment:
It's unclear to me why `http` is mentioned here. Shouldn't it be RESP(REdis Serialization Protocol)?

Reply:
Ah yes, so this is the core of the problem. 
The Recis URI is transmitted from the Prometheus server to the exporter as part of the http req when scraping the redis instance.
If you need to configure the exporter to use a password then use the command line flag (and yes, right now this will only allow to configure one password across all the instances you intend to scrape. if you need more than use more redis_exporters).

Reply:
See here for discussion in the original PR: https://github.com/oliver006/redis_exporter/pull/256#discussion_r283007985

Reply:
Oh I know what you mean, it's to remind users to not leak Redis password when scraping. Frankly I doubt in what degree it can prevent users from doing that. Careless people would configure URL with password regardless, those who do care can do things like putting exporter in controlled environment (we run it on the same machine of the Prometheus server), or adding a reverse proxy in front of exporter to terminate TLS.

I know our setup may not be a typical scenario, so I'm good to remove this change, but I feel the ability to batch scraping password protected instances can be useful to some others, so maybe we can see if there's a better idea.

Reply:
Right, you got it.
And yes, I there are ways to send an extra piece of config over from the prometheus server to the exporter using the `params` config of the Prometheus server but that's not implemented (yet).

Reply:
@oliver006 do you think this is okay? As you mentioned, even though Redis is encrypted, the password can be leaked if the user doesn't have security in mind. IMO a caution in README is sufficient as security is the users' responsibility.

Reply:
I'd remove this. Let's not let people pass a password over the wire. But let's keep your cert changes, they're good.